cluster-autoscaler/expander/grpcplugin/README.md

```diff
@@ -16,12 +16,12 @@ There are a wide variety of use cases here. Some examples are as follows:
 ## Configuration options
 As using this expander requires communication with another service, users must specify a few options as CLI arguments.
 
-```yaml
---grpcExpanderUrl
+```bash
+--grpc-expander-url
 ```
 URL of the gRPC Expander server, for CA to communicate with.
-```yaml
---grpcExpanderCert
+```bash
+--grpc-expander-cert
```

Comment:
you will also need to update these args below on line 35, that line says `grpcExapnderURl` and  `grpcExpanderCert`